exporter/containerimage/writer.go

```diff
@@ -947,3 +947,22 @@ func getRefMetadata(ref cache.ImmutableRef, limit int) []refMetadata {
 	}
 	return metas
 }
+
+// bytesReader is an alternative version of the bytes.Reader that implements
```

Comment:
The comment is from the previous implementation

Reply:
Fixed.